### ISSUE TITLE
Failures for 2019-03-06 pre-prod XML 3 of 5

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#1533]
pre-prod XML batch 2019-03-06 3 of 5:
https://drive.google.com/open?id=1qWHKx21dqiiQ6FlsazFhDdEYzRy_ebE4

Errors during e-QIP validation:
```
ARN[0002] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S14-SelectiveServiceRegistration: Answer "Were you born a male after December 31, 1959?" question yes or no. 
WARN[0002] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S16-PersonalReferences: Provide one or more valid telephone numbers. 
WARN[0002] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S16-PersonalReferences: Provide one or more valid telephone numbers. 
WARN[0002] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S16-PersonalReferences: Provide one or more valid telephone numbers. 
```

**Comment by ryanhofdotgov** [comment]
Selective service error looks unique – others are duplicate in the other pre-prod XML issues.

**Comment by ryanhofdotgov** [comment]
Exceptions 2,3,4 are duplicates of the one on #1516. 

Exception 1 does not appear to be an XML generation issue. It appears to be caused by the front end permitting a form to be submitted without a user providing any data for the Selective Service Section or possibly a scenario where the front-end becomes out-of-sync with the backend. 

I've attempted to reproduce the Exception 1 symptoms but without success – I've tried creating an application from scratch, skipping the section, going to over-all review, then back, etc. In this way, it is similar to https://github.com/18F/e-QIP-prototype/issues/1509

Closing as `duplicate` and `can't reproduce`.
